metrics/default-mobile-app-metrics.md

```diff
@@ -0,0 +1,142 @@
+# Abstract
+
+This document contains the proposal for gathering some default metrics of mobile apps and making them available to developers.
+
+## Goals
+
+### Goals
+
+- Provide mobile app developers a service that can receive some basic default metrics, namely sdk version, app version and platform version
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions
+
+### Non Goals
+
+- Identify individual devices or users in metrics events (bonus)
+
+- Registering of user-defined metrics and logs (not prioritized; just kept in mind)
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service (separate epic/proposal)
+
+## Problem Description
+
+As mobile apps are developed, improved and iterated on, different versions of those apps can be used on many different devices. 
+Metrics data becomes invaluable for developers to observe the landscape of application installations and make sure further 
+development and deployment of backend services will not affect users.
+
+## Proposed Solution
+
+A Metrics Service will be available from the Service Catalog, which can be provisioned by developers for use with their mobile applications.
+
+The following diagram illustrates the components:
+
+![service diagram](./default-mobile-metrics-system-landscape.png)
+
+### Mobile App Client
+
+Platform-specific SDK for targeting the metrics service will be available as IOS, Android and Cordova Plugins, React Native plugins.
+
+Upon installation into a project and providing configuration for targeting the Metrics Service the SDK will automatically 
+send the default metrics information for the application with a unique ID to the metrics service.
+This information will be sent to the service every time the application starts.  
+
+### Mobile Metrics Service
+
+A thin HTTP-based service will be present to receive metrics data from the mobile applications and then to forward to the storage service.
+
+At a PoC level, the endpoints available from the Mobile Metrics Service should be:
+
+`POST /default-metrics`
+
+Receives `Content-Type: application/json` containing JSON with following schema:
+```
+{
+  "clientId": "deadbeef123"
+  "appId": "com.example.someApp"
+  "sdkVersion": "2.4.6"
+  "appVersion": "256",
+  "platform": "android",
+  "platformVersion": "27",
+  "timestamp": "12223123123"
+}
+```
+
+Timestamp must be in UTC, without any time zone. Daylight saving should also be discarded. Server side application will get the timestamp and convert it to timestamp with server's timezone.
+
```

Comment:
I made the bold statements above about the timestamp, but would love to hear what you're going to say.